src/main/java/org/jabref/logic/autosaveandbackup/BackupManager.java

```diff
@@ -105,6 +105,16 @@ public static BackupManager start(BibDatabaseContext bibDatabaseContext, BibEntr
         return backupManager;
     }
 
+    /**
+     * Marks the backup as discarded at the library which is associated with the given {@link BibDatabaseContext}.
+     *
+     * @param bibDatabaseContext Associated {@link BibDatabaseContext}
+     */
+    public static void discardBackup(BibDatabaseContext bibDatabaseContext) {
+        runningInstances.stream().filter(instance -> instance.bibDatabaseContext == bibDatabaseContext).forEach(
```

Comment:
Why not equals?

Reply:
We copied and pasted the other working code.

Reply:
fair enough